src/Kunstmaan/NodeSearchBundle/EventListener/NodeIndexUpdateEventListener.php

```diff
@@ -18,17 +20,23 @@ class NodeIndexUpdateEventListener implements NodeIndexUpdateEventListenerInterf
     /** @var ContainerInterface */
     private $container;
 
+    /** @var EntityManagerInterface */
+    private $em;
+
     /** @var NodePagesConfiguration */
     private $nodePagesConfiguration;
 
     /** @var array */
     private $entityChangeSet;
 
-    /**
-     * @param ContainerInterface $container
-     */
-    public function __construct(/* NodePagesConfiguration */ $nodePagesConfiguration)
+    public function __construct(/** NodePagesConfiguration */
+        $nodePagesConfiguration, /** EntityManagerInterface */
+        $em = null)
     {
+        if (null !== $em) {
```

Comment:
The null check is not needed. We can just assign the value as we check the property in the method below.